pubspec.yaml

```diff
@@ -128,6 +128,10 @@ dependency_overrides:
     git:
       url: https://github.com/pedrox-hs/flutter_tags.git
       ref: null-safety
+  mocktail_image_network:
+    git:
+      url: https://github.com/pedrox-hs/mocktail.git
+      path: packages/mocktail_image_network
```

Comment:
Adicionei esse override temporariamente, dependendo de como for resolvido o PR https://github.com/felangel/mocktail/pull/232 irei remover essas linhas e se caso não for aprovado talvez implementar uma outra estratégia para telas que contem SVG remotos.